domain/src/main/kotlin/feature/BackupManager.kt

```diff
@@ -0,0 +1,58 @@
+package ltd.evilcorp.domain.feature
```

Comment:
Can you add a copyright header to this file? Either with yourself as the author, or if you don't want that, "aTox contributors" like in most other files.